scss/_regions/_navigation.scss

```diff
@@ -285,10 +285,83 @@
   a {
     display: block;
     font-weight: bold;
-    margin-left: 18px;
     padding: 6px 9px;
   }
 
+  .navigation__dropdown {
+
+    padding-top: 0;
+
+    @include media($tablet) {
+      padding-top: 12px;
+    }
+
+    ul {
+      visibility: visibile;
+      margin-top: 0;
+      @include media($tablet) {
+        visibility: hidden;
+      }
+
+      li {
+        padding: 0; 
+
+        @include media($tablet) {
+          padding: gutter() 0;
+        }
+      }
+
+    }
+
+    p {
+      visibility: hidden;
+      @include media($tablet) {
+        visibility: visible;
+        color: #fff;
+        padding-left: 8px;
+        padding-right: 8px;        
+      }      
+    }
+
+  }
+
+  .navigation__dropdown.clicked {
```

Comment:
Can you change `.clicked` to `.is-visible`? We use `is-` or `has-` prefixes for state classes (although it's not very well documented).